acj/static/modules/common/avatar-directive.js

```diff
@@ -6,12 +6,13 @@
 			scope: {
 				userId: '=',
 				avatar: '=',
+                forceDefault: '=?',
 				displayName: '=?',
 				fullName: '=?',
 				me: '=?'
 			},
 			template: '<a ng-href="#/user/{{ userId }}">' +
-						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro" /> ' +
+						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro{{ forceDefault ? \'&f=y\' : \'\' }}" /> ' +
```

Comment:
is it possible to collide with other people's avatar? I don't mind using one for all users (we can also set it to ComPAIR icon on gravatar, if we have one in the future)